src/execution_engines/Engine.ts

```diff
@@ -0,0 +1,27 @@
+export interface LoadContractIntoEngine {
+  contractAddress: string
+  executable: string
+}
+
+export interface UnloadContractFromEngine {
+  contractAddress: string
+}
+
+export interface ExecuteContract {
+  contractAddress: string
+  method: string
+  methodArgs: any
+}
+
+export type SmartContractResponse = any
+
+export interface Engine {
```

Comment:
We've got [a collision](https://github.com/input-output-hk/smart-contract-backend/blob/server-remodel/src/server/core/Engine.ts) with terminology. If we stick with the more explicit _Execution Engine_, then it will be easier to understand

```suggestion
export interface ExecutionEngine {
```